aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -254,6 +266,15 @@
    */
   public static final String S3_WRITE_TAGS_PREFIX = "s3.write.tags.";
 
+  /**
+   * Used by {@link S3FileIO} for access point. To set, we can pass a catalog property.
```

Comment:
I'm not sure what this is saying. Do you mean "prefix used for access point configuration"?